parm/config/config.base.emc.dyn

```diff
@@ -63,6 +63,8 @@ export DO_VRFY="YES"       # VRFY step
 #  use RUNMOS flag (currently in config.vrfy)
 export REALTIME="YES"
 
+#export DO_EFSOI="YES"
```

Comment:
This should be uncommented and NO. User can then change it to YES if they want to turn EFSOI on.